cmd/monaco/download/download_command.go

```diff
@@ -85,7 +85,16 @@ func GetDownloadCommand(fs afero.Fs, command Command) (cmd *cobra.Command) {
 		getDownloadEntitiesCommand(fs, command, cmd)
 	}
 
-	err := cmd.RegisterFlagCompletionFunc("api", completion.AllAvailableApis)
+	err := errors.Join(
```

Comment:
I am totally for `errors.Join`!
But it have side effect on our pretty print. 
e.g.
```
2023/04/06 08:45:37 ERROR the content of the environment variable "bla" is not set
2023/04/06 08:45:37 ERROR the content of the environment variable "adf" is not set
2023/04/06 08:45:37 ERROR the content of the environment variable "adsf" is not set
```
vs
```
2023/04/06 08:45:15 ERROR the content of the environment variable "bla" is not set
the content of the environment variable "adf" is not set
the content of the environment variable "adsf" is not set
```

Reply:
This is probably not an issue here, but in general...

Reply:
As discussed offline: errors.Join is, overall not the way to go. Here is an exception to the rule, as this code fails before the cli is basically started. That should never happen as we also have build targets to start the cli and check that no errors occur.